docs/essentials/exceptions/exceptions-best-practices.md

```diff
@@ -226,7 +117,7 @@ The following list contains guidelines for creating your own exceptions and when
 
 - Use grammatically correct error messages, including ending punctuation. Each sentence in a description string of an exception should end in a period. For example, "The log table has overflowed.” would be an appropriate description string.
 
-- Provide [Exception](https://msdn.microsoft.com/library/system.exception) properties for programmatic access. Provide additional properties for an exception (in addition to the description string) only when there's a programmatic scenario where the additional information is useful. For example, the [FileNotFoundException](https://msdn.microsoft.com/library/system.io.filenotfoundexception) provides the [FileName](https://msdn.microsoft.com/library/system.io.filenotfoundexception.filename) property.
+- Provide [Exception](http://dotnet.github.io/api/System.Exception.html) properties for programmatic access. Provide additional properties for an exception (in addition to the description string) only when there's a programmatic scenario where the additional information is useful. For example, the [FileNotFoundException](http://dotnet.github.io/api/System.IO.FileNotFoundException.html) provides the [FileName](http://dotnet.github.io/api/System.IO.FileNotFoundException.html#System_IO_FileNotFoundException_FileName) property.
```

Comment:
This is somewhat in conflict with "don't define new exception types unless you need to".